samcli/lib/utils/subprocess_utils.py

```diff
@@ -3,15 +3,19 @@
 """
 import logging
 import os
+import platform
 import sys
 from concurrent.futures.thread import ThreadPoolExecutor
-from subprocess import PIPE, Popen
+from subprocess import PIPE, STDOUT, Popen
 from time import sleep
 from typing import Any, AnyStr, Callable, Dict, Optional, Union
 
 from samcli.commands.exceptions import UserException
 from samcli.lib.utils.stream_writer import StreamWriter
 
+TERRAFORM_ERROR_PREFIX = [27, 91, 51, 49]
```

Comment:
Are these documented somewhere?

Reply:
No, I depended on what returned from terraform while testing

Reply:
Can we add some documentation about what are their meaning and where we can find more information?

Reply:
done